src/Tizen.TV.UIControls.Forms/RecycleItemsView.cs

```diff
@@ -312,6 +326,24 @@ public ScrollBarVisibility ScrollBarVisibility
             set { SetValue(ScrollBarVisibilityProperty, value); }
         }
 
+        /// <summary>
+        /// Gets or sets the focusable on header
+        /// </summary>
+        public bool FocusableOnHeader
```

Comment:
Ok, I also think AllowFocusXXX is better, I will update to AllowFocusXXX, 
I do not agree to single property, because it is independent property for each others.